Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,327 @@
+<?php
+
+/**
+ * File containing the main context class for PlatformUI.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context;
+
+use EzSystems\BehatBundle\Context\Browser\Context;
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert as Assertion;
+
+class PlatformUI extends Context
```

Comment:
Please add a summary of what the class does, a couple lines (it looks like an important one).